tests/phpunit/Civi/Core/SettingsManagerTest.php

```diff
@@ -50,7 +50,7 @@ public function testTwoDomains(): void {
     $da = \CRM_Core_DAO::createTestObject('CRM_Core_DAO_Domain');
     $db = \CRM_Core_DAO::createTestObject('CRM_Core_DAO_Domain');
```

Comment:
(*This comment has nothing to do with this line of code. But I want to put my comments as resolvable subthreads, and it has to do with testing, so... it goes on a test file.* 🙃)

The upgrade test failure seems to be pretty consistent.  The errors aren't immediately visible in the test-console, but you can find see them in the "Test report", eg [here](https://test.civicrm.org/job/CiviCRM-Test/18445/testReport/). (FWIW, I get [similar results](https://gist.github.com/totten/66ed9f1e0e37e205bab6b93880f62908).)

To reproduce it, one can execute:

```
civibuild ut dmaster '5.13.3*'
```

(That command basically just loads https://github.com/civicrm/civicrm-upgrade-test/blob/master/databases/5.13.3-multilingual_af_bg_en.mysql.bz2 into CiviCRM's MySQL database and runs `cv updb`.)

Reply:
Yep I clicked "retry" on that in vain. It's weird to me... but definitely a thread worth pulling. I'll have a look. Thanks for the pointers :+1: